server/translations.json

```diff
@@ -375,5 +375,43 @@
         "named_private_board_description": "您还可以创建一个命名的私有白板，它有一个自定义的URL，所有知道它名字的人都可以访问它。",
         "board_name_placeholder": "白板名称",
         "view_source": "GitHub上的源代码"
+    },
+ "vn": {
+        "hand": "Tay",
+        "loading": "Đang tải",
+        "tagline": "Một công cụ vẽ cộng tác trực tuyến miễn phí và mã nguồn mở. Cùng nhau phác thảo những ý tưởng mới trên WBO!",
+        "configuration": "Cấu hình",
+        "collaborative_whiteboard": "Bảng trắng cộng tác",
+        "size": "Size",
+        "zoom": "Zoom",
+        "tools": "Công cụ",
+        "rectangle": "Chữ nhật",
+        "square": "Vuông",
+        "circle": "Tròn",
+        "ellipse": "Hình elip",
+        "click_to_toggle": "Bật/tắt",
+        "menu": "Menu",
+        "text": "Text",
+        "mover": "Di chuyển",
+        "straight_line": "Đường thẳng",
+        "pencil": "Gạch ngang",
+        "grid": "Grid",
```

Comment:
Is this Vietnamese ?

Reply:
lưới ?

Reply:
In Vietnam, I am used to using common languages ​​in English.

Reply:
Is "grid" a common word in vietnamese ? If it isn't, then we should translate it.

Reply:
It is a very popular word. If put from the "lưới", it will be confusing.